Contracts/Elasticsearch/Search/SearchRequestInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CommonBundle\Contracts\Elasticsearch\Search;
+
+interface SearchRequestInterface
+{
+    public function addIndex(string $index): SearchRequestInterface;
+    public function addContentType(string $contentType): SearchRequestInterface;
+    public function addSourceInclude(string $include): SearchRequestInterface;
+    public function addSourceExclude(string $exclude): SearchRequestInterface;
+
+    /**
+     * @internal should only be used by the client
```

Comment:
had to Google this tag. Should the comment be more explicit that no BC promise is given on this method?